java-api-core/src/com/kinvey/java/store/requests/data/PushRequest.java

```diff
@@ -81,7 +86,16 @@ public Void execute() throws IOException {
                     }
                 }
                 try {
-                    syncManager.executeRequest(client, syncRequest);
+                    if (httpVerb == SyncRequest.HttpVerb.POST) {
+                        String tempID = syncRequest.getEntityID().id;
+                        GenericJson result = syncManager.executeRequest(client, syncRequest);
+                        T temp = cache.get(tempID);
+                        temp.set("_id", result.get("_id"));
```

Comment:
Would be good to use a constant string value here for `_id`, such as `Constants._ID`.

Reply:
I agree. It was updated.